packages/flutter/lib/src/foundation/memory_allocations.dart

```diff
@@ -271,7 +318,11 @@ class FlutterMemoryAllocations {
 
   /// Create [ObjectCreated] and invoke [dispatchObjectEvent] if there are listeners.
   ///
-  /// This method is more efficient than [dispatchObjectEvent] if the event object is not created yet.
+  /// If invoked in `builder` for [exemptFromRelease],
+  /// [ObjectCreated.isExemptFromDisposal] will be set to true.
```

Comment:
Is this comment supposed to be on this method?

Reply:
Those who invoke `dispatchObjectCreated` should know how exemptFromRelease is set.